### PR TITLE
Update list of datasets

### PR DESCRIPTION
**Pull request by tlozoot**
* Document the existence of several new adaptive datasets (HTTP requests, Load Balancing, LB Health Checks, Browser Insights)
* Deprecate some non-adaptive variants of these older datasets
* Remove references to deleted datasets

### CODE REVIEW

File: products/analytics/src/content/graphql-api/features/data-sets/index.md [hunk 1]
```diff
@@ -25,13 +25,13 @@ The following data sets (and associated nodes) are available in Cloudflare Analy
...
 | :-------------------------- | :----------------------------------------------------------------------------------------------------------------------------- |
-| Browser Insights            | `browserPerf1mGroups`                                                                                                          |
+| Browser Insights            | `browserInsightsAdaptiveGroups` `webVitalsAdaptiveGroups`                                                                      |
 | Firewall Activity Log       | `firewallEventsAdaptive` `firewallEventsAdaptiveByTimeGroups`                                                                  |
 | Firewall Analytics          | `firewallEventsAdaptiveGroups`                                                                                                 |
-| Health Check Analytics      | `healthCheckEvents` `healthCheckEventsGroups`                                                                                  |
-| HTTP Requests               | `httpRequests1mGroups` `httpRequests1hGroups` `httpRequests1dGroups` `httpRequests1mByColoGroups` `httpRequests1dByColoGroups` |
```
Comment (victor-perov) [inline products/analytics/src/content/graphql-api/features/data-sets/index.md:32]: I think we agreed to deprecate `httpRequests1[md]ByColoGroups`. Should we use put them here in that case?

File: products/analytics/src/content/graphql-api/features/data-sets/index.md [hunk 2]
```diff
@@ -57,12 +56,14 @@ The following data nodes are deprecated. To avoid disruption, migrate to replace
...
-| `firewallEventsByTimeGroups` | `firewallEventsByTimeGroupsAdaptive` | Oct 26, 2020 |
-| `firewallRulePreviewGroups`  | `httpRequestAdaptiveGroups`          | Oct 26, 2020 |
+| Node                         | Replacement node                     | Sunset date   |
+| ---------------------------- | ------------------------------------ | ------------- |
+| `httpRequestsCacheGroups`    | `httpRequestsAdaptiveGroups`         | March 1, 2021 |
+| `firewallRulePreviewGroups`  | `httpRequestsAdaptiveGroups`         | March 1, 2021 |
+| `healthCheckEvents`          | `healthCheckEventsAdaptive`          | March 1, 2021 |   
+| `healthCheckEventsGroups`    | `healthCheckEventsAdaptiveGroups`    | March 1, 2021 |  
+| `loadBalancingRequests`      | `loadBalancingRequestsAdaptive`      | March 1, 2021 | 
+| `loadBalancingRequestsGroups`| `loadBalancingRequestsAdaptiveGroups`| March 1, 2021 |
```
Comment (victor-perov) [inline products/analytics/src/content/graphql-api/features/data-sets/index.md:69]: Should we add `httpRequest1[md]ByColoGroups` here (replacement Node `httpRequestsAdaptiveGroups`)?

**Review by victor-perov — CHANGES_REQUESTED** [review]

**Review by soheiokamoto — APPROVED** [review]

File: products/analytics/src/content/graphql-api/features/data-sets/index.md (hunk 2 above)
Comment (tlozoot) [reply products/analytics/src/content/graphql-api/features/data-sets/index.md:69]: Yep, updated.

File: products/analytics/src/content/graphql-api/features/data-sets/index.md (hunk 1 above)
Comment (tlozoot) [reply products/analytics/src/content/graphql-api/features/data-sets/index.md:32]: Yes, updated.

**Review by victor-perov — APPROVED** [review]

**Pull request merged**